xmtp_mls/src/utils/topic.rs

```diff
@@ -1,5 +1,11 @@
 pub fn get_group_topic(group_id: &Vec<u8>) -> String {
-    format!("/xmtp/3/g-{}/proto", hex::encode(group_id))
+    format!("/xmtp/3/g-{}/proto", serialize_group_id(group_id))
+}
+
+pub fn serialize_group_id(group_id: &[u8]) -> String {
+    // TODO: I wonder if we really want to be base64 encoding this or if we can treat it as a
```

Comment:
The nodes treat it as a string, so it's going to have to get serialized somewhere.

Reply:
I copied this comment from elsewhere - I think it's musing over whether base64 would be more efficient than hex